data/json/items/chemicals_and_resources.json

```diff
@@ -367,7 +367,7 @@
     "description" : "Several crystals of aluminium sulphate. Historically, an agent for purifying water and as molluscicide, due to the fact that it degenerates their slime. Just before the Cataclysm, its main uses were for adjusting the acidity of the soil and as an additive to baking powder. If mixed with a strong acid, it could be used to saturate the soil with aluminium, causing all but the most resistant of plants to wither and die.",
     "material" : "powder",
     "volume" : 1,
-    "weight" : 350,
+    "weight" : 13.35,
```

Comment:
> C:DDA must not handle decimals too well.

It does handle them. However, the item weight is loaded and stored as integer. Everything behind the '.' is effectively ignored, sorry.